firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Protocols/chunks.txt

```diff
@@ -0,0 +1,262 @@
+/* chunks
+
+Scenario: A ChunkSplitter constructed with default parameters correctly handles input chunks of acceptable lengths
+
+  GIVEN("A char chunk splitter with an empty internal buffer of 102 bytes")
+  // assertions:
+  // output buffer is empty
+
+    WHEN("102 non-delimiter bytes are passed as input, and 103rd byte is passed as a delimiter, and output is called between each input")
+      THEN("The input method reports ok status for all the 102 non-delimeter bytes")
+      THEN("After the input of 102 non-delimiter bytes, The output method reports waiting status")
+      THEN("The input method reports output_ready status on the 103rd byte")
+      THEN("After the delimiter is passed to input, The output method reports ok status")
+      THEN("The output buffer is as expected")
```

Comment:
For all tests in this file, I would suggest specifying the length of the expected output buffer, e.g. THEN the output buffer has the expected sequence of 102 bytes. I think this will help make clear what the contents are, in the same way that the tests for CRCElements explicitly wrote out the expected bytes for short bodies/payloads.